API.md

```diff
@@ -801,7 +783,7 @@ const ManagedTableAccordionGroup = manageState(TableAccordionGroup, {
 Prop | Required | Default | Type | Description
 :--- | :------- | :------ | :--- | :----------
  `changeOnTitleClick` |  | ```false``` | bool | 
- `children` | true | `````` | node | 
+ `children` |  | ```undefined``` | node | 
```

Comment:
also odd this change here too - i'm wondering if `undefined` is exactly what we want to be showing here as the `default` value for this one, and the one above

Reply:
If provided apparently this features as a child and the counter is no longer deemed empty.